FreeRTOS/Demo/CORTEX_M3_MPS2_QEMU_GCC/FreeRTOSConfig.h

```diff
@@ -1,5 +1,5 @@
 /*
- * FreeRTOS V202012.00
+ * FreeRTOS V202011.00
```

Comment:
This is wrong

Reply:
yeah i fixed the checker, as it was complaining about 202012 and wants it 202011